src/ops/embedding.cc

```diff
@@ -14,6 +14,7 @@
  */
```

Comment:
I think git's a bit confused about this file, can you resolve the merge conflicts and then I'll take another look at it?